### ISSUE TITLE
display list got segmentation fault when Air Play to to another Mac

### ISSUE THREAD

**Issue by ligolas** [waydabber/m1ddc#45]
[1]    4525 segmentation fault  ./m1ddc display list

**Comment by ilgoncalves** [comment]
I’m having the same problem. I usually work with an external monitor and use Continuity with the iPad Pro (formerly Sidecar), which makes the iPad function as an external monitor as well.

<img width="303" alt="Screenshot 2024-08-16 at 15 45 25" src="https://github.com/user-attachments/assets/80d67d7b-76c0-473a-8e68-cca238361cd3">
<img width="459" alt="Screenshot 2024-08-16 at 15 51 24" src="https://github.com/user-attachments/assets/9f5d316a-b60e-4aaf-ba34-8d91f3736034">


When I disconnect the MacBook from the iPad, it works normally.

<img width="298" alt="Screenshot 2024-08-16 at 15 52 01" src="https://github.com/user-attachments/assets/7737bfdb-7347-4933-9b11-a1782e23c485">

<img width="591" alt="Screenshot 2024-08-16 at 15 46 53" src="https://github.com/user-attachments/assets/94b1b545-1afe-426e-8950-4a2e690bd38e">

@waydabber, maybe this isn’t set up to handle this type of monitor. Could we disable it?

**Comment by waydabber** [comment]
Yes, apparently this does not work well indeed. For more advanced CLI based DDC control you can try `betterdisplaycli` - it also has full CLI control to connect/disconnect Sidecar displays as well.

**Comment by ilgoncalves** [comment]
> Yes, apparently this does not work well indeed. For more advanced CLI based DDC control you can try `betterdisplaycli` - it also has full CLI control to connect/disconnect Sidecar displays as well.

Thank you for the quick response. I installed the Better Display app, but I couldn’t find `betterdisplaycli` on Homebrew. How can I install it?

**Comment by waydabber** [comment]
`brew install waydabber/betterdisplay/betterdisplaycli`

**Comment by cuddeford** [comment]
PR #55 aims to fix this. Works for me.
